x/estaking/keeper/elys_staked.go

```diff
@@ -1,42 +1,44 @@
 package keeper
 
 import (
-	"cosmossdk.io/math"
 	"github.com/cosmos/cosmos-sdk/store/prefix"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/elys-network/elys/x/estaking/types"
 )
 
 // SetElysStaked set a specific elysStaked in the store from its index
 func (k Keeper) SetElysStaked(ctx sdk.Context, elysStaked types.ElysStaked) {
-	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefix(types.ElysStakedKeyPrefix))
+	store := ctx.KVStore(k.storeKey)
 	b := k.cdc.MustMarshal(&elysStaked)
-	store.Set(types.ElysStakedKey(elysStaked.Address), b)
+	key := types.GetElysStakedKey(elysStaked.GetAccountAddress())
+	store.Set(key, b)
 }
 
 // GetElysStaked returns a elysStaked from its index
-func (k Keeper) GetElysStaked(ctx sdk.Context, address string) math.Int {
-	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefix(types.ElysStakedKeyPrefix))
-
-	b := store.Get(types.ElysStakedKey(address))
-	if b == nil {
-		return math.ZeroInt()
+func (k Keeper) GetElysStaked(ctx sdk.Context, address sdk.AccAddress) types.ElysStaked {
+	key := types.GetElysStakedKey(address)
+	store := ctx.KVStore(k.storeKey)
+	bz := store.Get(key)
+	if bz == nil {
+		return types.ElysStaked{
```

Comment:
why do we use this data structure ? key is `address` and we add it again in value ?

Reply:
If we don't it won't get exported in the export genesis and no way to know key while importing genesis